components/brave_wallet_ui/panel/container.tsx

```diff
@@ -465,6 +465,10 @@ function Container (props: Props) {
     // Logic here to Cancel Adding a Suggested Token
   }
 
+  const onAddNetwork = () => {
+    props.walletPanelActions.expandWalletNetworks()
```

Comment:
Instead of having a duplicated actions, we could do
```
    props.walletActions.expandWalletNetworks()
```

Reply:
Thank you, done